css/style.css

```
@@ -0,0 +1,184 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+html {    
+    font-size: 100%;
+    overflow-x: hidden;
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.header {
+    position: absolute;
```

Let's use 2 spaces indents. Here and everywhere
```suggestion
.header {
  position: absolute;
```